cmd/utils/flags.go

```diff
@@ -1104,8 +1095,6 @@ func setBootstrapNodes(ctx *cli.Context, cfg *p2p.Config) {
 		urls = params.GoerliBootnodes
 	case ctx.Bool(KilnFlag.Name):
 		urls = params.KilnBootnodes*/
-	case ctx.Bool(LveFlag.Name):
-		urls = params.LveBootnodes
 	case ctx.Bool(SeoulFlag.Name):
 		urls = params.SeoulBootnodes
 	case ctx.Bool(GwangjuFlag.Name):
```

Comment:
Annapurna flag?

Reply:
The flag refers to the name of the network, so it is not needed.